### PR TITLE
1 0 audio media

### PR DESCRIPTION
**Pull request by itkg-ngilain**
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/45
https://github.com/open-orchestra/open-orchestra-media-bundle/pull/126

### CODE REVIEW

File: MediaBundle/Resources/views/BBcode/front_audio.html.twig [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<audio controls>
+    <source src="{{ media_url}}" type="{{ media_type }}">
+    {{ 'open_orchestra_media.strategy.audio.tag_not_supported'|trans }}
+</audio>
```
Comment (itkg-nanne) [inline MediaBundle/Resources/views/BBcode/front_audio.html.twig]: return

File: MediaBundle/Resources/views/BBcode/wysiwyg_audio.html.twig [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<img
+    class="tinymce-audio"
+    src="{% image '@OpenOrchestraMediaAdminBundle/Resources/public/images/icon/tinymce-audio.png' %}{{ asset_url }}{% endimage %}"
+    data-id="{{ media_id }}"
+/>
```
Comment (itkg-nanne) [inline MediaBundle/Resources/views/BBcode/wysiwyg_audio.html.twig]: return

File: MediaBundle/Resources/views/BBcode/front_audio.html.twig (hunk 1 above)
Comment (itkg-alavieille) [inline MediaBundle/Resources/views/BBcode/front_audio.html.twig]: eof

File: MediaBundle/Resources/views/BBcode/wysiwyg_audio.html.twig (hunk 2 above)
Comment (itkg-alavieille) [inline MediaBundle/Resources/views/BBcode/wysiwyg_audio.html.twig]: eof

File: Media/DisplayMedia/Strategies/AbstractStrategy.php [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 namespace OpenOrchestra\Media\DisplayMedia\Strategies;
 
+use Symfony\Component\DependencyInjection\ContainerAware;
 use OpenOrchestra\Media\DisplayMedia\DisplayMediaInterface;
 use OpenOrchestra\Media\Model\MediaInterface;
 use Symfony\Component\HttpFoundation\RequestStack;
```
Comment (itkg-alavieille) [inline Media/DisplayMedia/Strategies/AbstractStrategy.php]: this use is never used

File: Media/DisplayMedia/Strategies/AudioStrategy.php [hunk 4]
```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace OpenOrchestra\Media\DisplayMedia\Strategies;
+
+use OpenOrchestra\Media\Model\MediaInterface;
+use Symfony\Component\HttpFoundation\RequestStack;
```
Comment (itkg-alavieille) [inline Media/DisplayMedia/Strategies/AudioStrategy.php]: this use is never used

**Comment by itkg-ngilain** [comment]
@itkg-alavieille , @itkg-nanne : update done

**Comment by itkg-alavieille** [comment]
:+1:

**Pull request merged**